apps/financial-aid/web-osk/src/components/Logo/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import cn from 'classnames'
+
+import LogoSvg from './LogoSvg'
+import LogoSIS from './LogoSIS'
+
+interface LogoProps {
+  className?: string
+  customLogo?: boolean
+}
+
+const Logo = ({ className, customLogo }: LogoProps) => {
+  return (
+    <>
+      {customLogo ? (
+        <a
+          href="https://www.hafnarfjordur.is/"
+          target="_blank"
+          className={cn({ [`${className}`]: true })}
+        >
+          <LogoSvg />
+        </a>
+      ) : (
+        <a
+          href="https://www.samband.is/"
+          target="_blank"
+          className={cn({ [`${className}`]: true })}
+        >
+          <LogoSIS />
+        </a>
+      )}
+    </>
+  )
+}
```

Comment:
In this PR, we are fetcing the munacipilaty and displaying correct logo and homepage depending on values: https://github.com/island-is/island.is/pull/5044/files#diff-e2064a8589f84aa1ffa7f90b3b6d95a8e30b79aada5b1993a63e13500ff34a96R19

Should we do the same here? And skip having a customLogo param and have default values instead?

Reply:
Well spotted, yess! 💯